split.Console/Models/Blackjack/BlackjackGame.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackGame : Game
+    {
+        Deck CardDeck;
+        BlackjackPlayer Dealer;
+
+        public BlackjackGame(int decks) : base(GameType.Blackjack)
+        {
+            CardDeck = new Deck(decks);
+            Dealer = new BlackjackPlayer("Dealer", null);
+        }
+
+        /// <summary>
+        /// Adds a newly created player to the game
+        /// </summary>
+        /// <param name="newPlayer">Created player to be added to the game</param>
+        public void AddPlayer(BlackjackPlayer newPlayer)
+        {
+            Players.Add(newPlayer);
```

Comment:
Just so you know, this probably won't compile. The compiler won't know what ```Players``` is.